bedrock/products/templates/products/vpn/resource-center/base-article.html

```diff
@@ -0,0 +1,86 @@
+{#
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% from 'macros-protocol.html' import split %}
+
+{% extends "products/vpn/base.html" %}
+
+{% block page_css %}
+  {{ css_bundle('vpn-resource-center')}}
+{% endblock page_css %}
+
+{% block page_title%}{% block article_title %}{% endblock article_title %}{% endblock page_title%}
+
+{% block content %}
+<ol class="mzp-c-breadcrumb">
+  <li class="mzp-c-breadcrumb-title">
+    <a class="mzp-c-breadcrumb-item-link" href="{{ url('products.vpn.resource-center.landing') }}">{{ ftl('vpn-resource-center-title') }}</a>
+  </li>
+  <li class="mzp-c-breadcrumb-item">
+    {{ self.article_title() }}
+  </li>
+</ol>
+<div class="vpn-article-container">
+  <div class="mzp-l-content mzp-t-content-md">
+      <article class="mzp-c-article vpn-article">
+        {# Categories will be used at some point but not yet #}
+        {# <p class="vpn-article-category">{% block article_category %}{{ self.article_category().capitalize() }}{% endblock %}</p> #}
+        <h2>{{ self.article_title() }}</h2>
+        {% block vpn_article %}{% endblock vpn_article %}
+      </article>
+    <div class="vpn-c-vote">
```

Comment:
Is it worth hiding this content if JS is not enabled?

Reply:
+1, would agree with hiding based on js availability, I think there's a `no-js` class that appears on the html tag, so we could `display:none` the vote section in those cases